CliClient/locales/ko.po

```diff
@@ -2928,7 +2906,7 @@ msgstr "변경 사항 버리기"
 #: /Users/tessus/data/work/joplin/Tools/../ReactNativeClient/lib/components/screens/note.js:130
 #, javascript-format
 msgid "No item with ID %s"
-msgstr "ID %s에 해당하는 항목이 없습니다"
+msgstr "계정 %s에 해당하는 항목이 없습니다"
 
```

Comment:
ID is a column name, thus must not be translated. in the cli version ID can be used to specify notes. It's more consistent not translating this part.